src/layouts/index.js

```diff
@@ -71,14 +71,18 @@ const HeaderDesktop = ({ language, hasLanguage, setLanguage }) => {
           <strong>eBPF Summit</strong>
           <Link className="link" to="/summit-2021">2021</Link>
           <Link className="link" to="/summit-2020">2020</Link>
+          <strong>eBPF Day (KubeCon)</strong>
```

Comment:
```suggestion
          <strong>Cloud Native eBPF Day (co-located with KubeCon)</strong>
```

Reply:
I kept it brief as then the whole menu does not look good anymore. I guess it's clear if someone clicks on the link, no?

Reply:
Maybe.  I haven't seen what it looks like with the wider look, so I'll take your point.   I was hoping the drop down would be wide but the label it dropped down from wouldnt be but I take it that's not the case :)   Still, "Cloud Native eBPF Day" is the correct name afaik, so if you have to make it shorter then
```suggestion
          <strong>Cloud Native eBPF Day</strong>
```

Reply:
Ok, done.

Reply:
I'd like to see a deploy preview with the wider labels, since in the browsers I've tried the drop down can be wider than the menu label it drops down from, so width shouldn't be an issue unless I'm missing something.   Should I generate a separate PR so the preview will be generated?